tgui/packages/tgui/interfaces/Pressurizer.js

```diff
@@ -0,0 +1,203 @@
+        <Flex>
+          <Flex.Item width="900px">
+            <PortableBasicInfo
+              connected={connected}
+              pressure={pressure}
+              maxPressure={maxPressure}>
+              <Divider />
+              <LabeledList>
+                <LabeledList.Item label="Emergency Blast Release" >
+                  <Button
+                    fluid
+                    textAlign="center"
+                    icon="circle"
+                    content={getArmedState()}
+                    disabled={pressure<maxPressure*0.2 || !airSafe}
+                    color={blastArmed ? 'bad' : 'average'}
+                    onClick={() => handleArmPressurizer()} />
+                </LabeledList.Item>
+                <LabeledList.Item label="Delay">
+                  <Button onClick={() => handleSetBlastDelay(minArmDelay)}>
+                    Min
+                  </Button>
+                  <NumberInput
+                    animated
+                    width="7em"
+                    value={blastDelay}
+                    minValue={minArmDelay}
+                    maxValue={maxArmDelay}
+                    onChange={(e, targetDelay) => handleSetBlastDelay(targetDelay)} />
+                  <Button onClick={() => handleSetBlastDelay(maxArmDelay)}>
+                    Max
+                  </Button>
+                </LabeledList.Item>
+              </LabeledList>
+              <Divider />
+              <LabeledList>
+                <LabeledList.Item label="Fan Status">
+                  <Button color={fanState === FanState.Off ? 'bad' : 'default'}
+                    onClick={() => handleSetFan(FanState.Off)}>
+                    Off
+                  </Button>
+                  <Button color={fanState === FanState.In ? 'good' : 'default'}
+                    onClick={() => handleSetFan(FanState.In)}>
+                    In
+                  </Button>
+                  <Button color={fanState === FanState.Out ? 'good' : 'default'}
+                    onClick={() => handleSetFan(FanState.Out)}>
+                    Out
+                  </Button>
+                </LabeledList.Item>
+                <LabeledList.Item label="Release pressure">
+                  <Button onClick={() => handleSetPressure(minRelease)}>
+                    Min
+                  </Button>
+                  <NumberInput
+                    animated
+                    width="7em"
+                    value={releasePressure}
+                    minValue={minRelease}
+                    maxValue={maxRelease}
+                    onChange={(e, targetPressure) => handleSetPressure(targetPressure)} />
+                  <Button onClick={() => handleSetPressure(maxRelease)}>
+                    Max
+                  </Button>
+                </LabeledList.Item>
+              </LabeledList>
+            </PortableBasicInfo>
+            <Section
+              title="Material Processing"
+              minHeight="90px"
```

Comment:
At one point I _think_ it was attempting to maintain a Section size that was too short horizontally because I was being used incorrectly during development.  At this point it does nothing.

Removed.